Gruntfile.coffee

```diff
@@ -56,51 +32,63 @@ module.exports = (grunt) ->
       options:
         configFile: 'coffeelint.json'
 
+    clean: ['./tmp']
+
+    coffee:
+      test:
+        expand: true,
+        cwd: './test/',
+        src: ['*.coffee'],
+        dest: './tmp/test/',
+        ext: '.js'
+
     mochaTest:
       native:
         options:
           reporter: 'spec'
           require: [
-            'coffee-script/register'
-            'coffee-coverage/register-istanbul'
+            ->
+              global._$coffeeIstanbul = {}
```

Comment:
This is ugly, but needed. `istanbul` add something like 

```js
if (typeof _$coffeeIstanbul === 'undefined') _$coffeeIstanbul = {};
```

in every modules. This breaks badly with `babel-register` which wrap the code in a function with a `use strict` pragma.